src/Rules/Resources/LocalizedControlTypeNames.resx

```diff
@@ -119,125 +119,166 @@
   </resheader>
   <data name="AppBar" xml:space="preserve">
     <value>app bar</value>
+    <comment>The localized type (role) of a UI Automation control, as provided by the system. If you know it, the localized name used by the UI Automation accessibility framework in the target language should be used.</comment>
```

Comment:
Out of scope for this PR, but I wonder if a tool exists (or could be created) to help localizers obtain this information directly from their system, rather than trying to hunt it down somewhere

Reply:
No idea. If I was localizing this, I'd check with NVDA (using the Python console to get the name directly from UIA in case NVDA overrides).